Assets/BossRoom/Scripts/Client/Game/Action/FXProjectileTargetedActionFX.cs

```diff
@@ -120,7 +120,7 @@ private NetworkObject GetTarget()
         /// </summary>
         private bool IsParentAnNPC()
         {
-            var targetable = m_Parent.Parent.GetComponent<ITargetable>();
```

Comment:
Why are we changing this from the clientvis Parent to the transform.Parent. From my understanding the Parent property of the client visualization is not the actual parent of the object? XML doc says:
```
        /// <summary>
        /// Returns our pseudo-Parent, the object that owns the visualization.
        /// (We don't have an actual transform parent because we're on a top-level GameObject.)
        /// </summary>
        public Transform Parent { get; private set; }
```

Reply:
It got changed some time ago with a refactor of the characters. Now the clientVis is a child of the actual character, so this property is no longer necessary.

Reply:
I believe you can retrieve that from `m_Parent.NetState.IsNpc`, right?

Reply:
I just noticed the removal of that property was not included in my commit. Fixed it now.

Reply:
Looks good to me. @LukeStampfli I'll let you resolve.